modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -2467,6 +2469,33 @@ private String _getIndexFieldName(String optionKey, String languageId) {
 				newURLTitleMap.put(
 					LocaleUtil.toLanguageId(titleEntry.getKey()), urlTitle);
 			}
+			else if ((urlTitle != null) && urlTitle.equals("")) {
```

Comment:
We should use `StringPool.BLANK`
```suggestion
			else if ((urlTitle != null) && urlTitle.equals(StringPool.BLANK)) {
```